app/views/categories/edit.html.erb

```diff
@@ -0,0 +1,14 @@
+<h1> Edit </h1>
+
+<%= form_with(model: @category, local: true) do |form| %>
```

Comment:
Cuando puedas migra todo el formulario a un parcial `_form.html.erb` en la misma carpeta porque así evitamos tener que repetir el código en dos vistas y el código se puede mantener mejor :wink:

Reply:
Ok!, I'm going to do it right now.